src/main/java/io/github/programminglife2016/pl1_2016/parser/nodes/NodeMap.java

```diff
@@ -24,19 +24,23 @@ public final String toJson() {
      */
     @Override
     public final void recalculatePositions() {
-        for(Map.Entry<Integer, Node> entry : entrySet()) {
+        for (Map.Entry<Integer, Node> entry : entrySet()) {
             Node node = entry.getValue();
             Node start = retrieveSegment(node, true);
             Node end = retrieveSegment(node, false);
-            int x = (start.getX() + end.getX())/2;
-            int y = (start.getY() + end.getY())/2;
+            int x = (start.getX() + end.getX()) / 2;
+            int y = (start.getY() + end.getY()) / 2;
             this.get(entry.getKey()).setXY(x, y);
         }
     }
 
```

Comment:
Is it really necessary to expand the if statement? The one-liner seemed to work just as well